app/data/flags.json

```diff
@@ -0,0 +1,48 @@
+{
+  "GEOLOCATION": {
+    "label": "Geolocation",
+    "defaultValue": false
+  },
+  "INFO_BUBBLE_HOVER_POLYGON": {
+    "label": "Info bubble — [debug] show Tognazzini zone",
+    "defaultValue": false
+  },
+  "LOCALES_LEVEL_1": {
+    "label": "Locales — level 1+ (in progress)",
+    "defaultValue": false,
+    "disabled": false
+  },
+  "LOCALES_LEVEL_2": {
+    "label": "Locales — level 2+ (complete, in testing)",
+    "defaultValue": false,
+    "disabled": false
+  },
+  "LOCALES_LEVEL_3": {
+    "label": "Locales — level 3 (complete, final)",
+    "defaultValue": false,
+    "disabled": false
+  },
+  "SEGMENT_FLEX_ZONE": {
+    "label": "Segment — flex zone",
+    "defaultValue": false,
+    "disabled": true
+  },
+  "SEGMENT_INCEPTION_TRAIN": {
+    "label": "Segment — the train from “Inception”",
+    "defaultValue": false,
+    "disabled": false
+  },
+  "SEGMENT_ANGLED_PARKING": {
+    "label": "Segment — angled parking",
+    "defaultValue": false,
```

Comment:
Are you working from the latest version of the master branch? For instance, the current `https://github.com/streetmix/streetmix/blob/master/assets/scripts/app/flag_data.js` has the `defaultValue` for `SEGMENT_ANGLED_PARKING` set to `true`. Please go through and verify that this json file matches all the properties in `flag_data.js` exactly.

To remove confusion, let's also remove `flag_data.js` after it's been ported here, then update any scripts that use it: https://github.com/streetmix/streetmix/search?q=flag_data&unscoped_q=flag_data

For now, just import the file directly when compiling the front-end, don't use the API yet.